src/main/res/values/string_pref_keys.xml

```diff
@@ -20,14 +20,28 @@
 
 <string name="pref_VALUE_xxx_yyy">value</string>
 <string name="pref_VALUE_xxx_zzz">value</string>
+
+<string name="pref_DESC_xxx">value</string>
 ...
 -->
 <resources>
+    <string name="pref_CAT_APPEARENCE" translatable="false">CAT_APPEARANCE</string>
+    <string name="pref_CAT_COMPASS" translatable="false">CAT_COMPASS</string>
+    <string name="pref_CAT_CREDENTIALS" translatable="false">CAT_CREDENTIALS</string>
+    <string name="pref_CAT_DIRECTIONS_1" translatable="false">CAT_DIRECTIONS_1</string>
+    <string name="pref_CAT_DIRECTIONS_2" translatable="false">CAT_DIRECTIONS_2</string>
+    <string name="pref_CAT_GLOBAL" translatable="false">CAT_GLOBAL</string>
+    <string name="pref_CAT_LOCALIZATION_1" translatable="false">CAT_LOCALIZATION_1</string>
+    <string name="pref_CAT_LOCALIZATION_2" translatable="false">CAT_LOCALIZATION_2</string>
+    <string name="pref_CAT_LOCALIZATION_3" translatable="false">CAT_LOCALIZATION_3</string>
+    <string name="pref_CAT_LOCATION_1" translatable="false">CAT_LOCATION_1</string>
+    <string name="pref_CAT_LOCATION_2" translatable="false">CAT_LOCATION_2</string>
     <string name="pref_KEY_X_ABOUT" translatable="false">KEY_X_ABOUT</string>
     <string name="pref_KEY_X_HIDDEN_PREFERENCES" translatable="false">KEY_X_HIDDEN_PREFERENCES</string>
     <string name="pref_KEY_X_LOGIN_PREFERENCES" translatable="false">KEY_X_LOGIN_PREFERENCES</string>
     <string name="pref_KEY_S_APPLICATION_VERSION_LAST" translatable="false">KEY_S_APPLICATION_VERSION_LAST</string>
     <string name="pref_KEY_S_ROOT" translatable="false">KEY_S_ROOT</string>
+    <string name="pref_DESC_ROOT">Current Folder: %1$s</string>
```

Comment:
Values in this file should all have the attribute `translatable="false"`

Reply:
But this is a user description and needs to be translatable.

Reply:
Then please move it to the normal strings file

Reply:
But why move something which is related to the settings out of the file which is created for settings... Sigh I'll do it after lunch...

Reply:
Done via 46fe213

Reply:
The descripting strings for preferences are defined in `strings_pref.xml`.
Please move there.

Reply:
Done